apps/tt/aat/base/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../../base
+  - ../../../rbac/nonprod-role.yaml
+namespace: tt
```

Comment:
Can you please create a more meaingful namespace? `tax-tribunals` instead of `tt`